python.d/proxysql.chart.py

```diff
@@ -0,0 +1,349 @@
+# -*- coding: utf-8 -*-
+# Description: Proxysql netdata python.d module
+# Author: Ali Borhani (alibo)
+# SPDX-License-Identifier: GPL-3.0+
+
+from bases.FrameworkServices.MySQLService import MySQLService
+
+# default module values (can be overridden per job in `config`)
+# update_every = 3
+priority = 60000
+retries = 60
+
+
+def query(table, *params):
+    return 'SELECT {params} FROM {table}'.format(table=table, params=', '.join(params))
+
+
+# https://github.com/sysown/proxysql/blob/master/doc/admin_tables.md#stats_mysql_global
+QUERY_GLOBAL = query("stats_mysql_global",
+                     "Variable_Name",
+                     "Variable_Value",
+                     )
+
+# https://github.com/sysown/proxysql/blob/master/doc/admin_tables.md#stats_mysql_connection_pool
+QUERY_CONNECTION_POOL = query("stats_mysql_connection_pool",
+                              "hostgroup",
+                              "srv_host",
+                              "srv_port",
+                              "status",
+                              "ConnUsed",
+                              "ConnFree",
+                              "ConnOK",
+                              "ConnERR",
+                              "Queries",
+                              "Bytes_data_sent",
+                              "Bytes_data_recv",
+                              "Latency_us",
+                              )
+
+# https://github.com/sysown/proxysql/blob/master/doc/admin_tables.md#stats_mysql_commands_counters
+QUERY_COMMANDS = query("stats_mysql_commands_counters",
+                       "Command",
+                       "Total_Time_us",
+                       "Total_cnt",
+                       "cnt_100us",
+                       "cnt_500us",
+                       "cnt_1ms",
+                       "cnt_5ms",
+                       "cnt_10ms",
+                       "cnt_50ms",
+                       "cnt_100ms",
+                       "cnt_500ms",
+                       "cnt_1s",
+                       "cnt_5s",
+                       "cnt_10s",
+                       "cnt_INFs",
+                       )
+
+GLOBAL_STATS = [
+    'client_connections_aborted',
+    'client_connections_connected',
+    'client_connections_created',
+    'client_connections_non_idle',
+    'proxysql_uptime',
+    'questions',
+    'slow_queries']
```

Comment:
Please format multi-line containers (lists, dicts, tuples, etc) like this so that they're consistent with the rest of our code:

```
GLOBAL_STATS = [
    'client_connections_aborted',
    'client_connections_connected',
    'client_connections_created',
    'client_connections_non_idle',
    'proxysql_uptime',
    'questions',
    'slow_queries'
]
```

Reply:
Done :)